documentation/docs/develop/02-extensions/09-api-changes/0.9.mdx

```diff
@@ -45,3 +45,88 @@ Dispatchers.UntickedAsync.launch {}
 ```
 
 If working in **IntelliJ**, it will automatically suggest the correct replacements.
+
+## `SpeakerEntry` and `EntityInstanceEntry` Changes
+
+The way sound is handled in `SpeakerEntry` and, by extension, `EntityInstanceEntry` has been updated to use variables.
+
+### Sound Field
+
+The `sound` field now utilizes a `Var<Sound>` type. This change affects how you set and get sound values.
+
+```kotlin showLineNumbers
+// highlight-red
+class CustomEntityDefinition(
+    ...
+    override val sound: Sound = Sound.EMPTY,
+    ...
+) : SimpleEntityDefinition {
+// highlight-green
+class CustomEntityDefinition(
+    ...
+    override val sound: Var<Sound> = ConstVar(Sound.EMPTY),
+    ...
+) : SimpleEntityDefinition {
+```
+
+**To get the sound:**
+Retrieving the sound now requires the context of a `player` and an optional `interactionContext`.
+
+```kotlin showLineNumbers
+// highlight-red
+val sound: Sound = entry.sound
+// highlight-green
+val sound: Sound = entry.sound.get(player, interactionContext?)
+```
+
+## `Sound` Class Changes
+
+Several fields and methods within the `Sound` class have been updated.
+
+### Volume and Pitch Fields
+
+The `volume` and `pitch` fields have been changed from `Float` to `Var<Float>`.
+
+**To set the volume and pitch:**
+When creating a new `Sound` instance, you now need to wrap the `volume` and `pitch` values in a `Var`, such as `ConstVar`.
+
+```kotlin showLineNumbers
+// highlight-red
+val sound = Sound(DefaultSoundId("minecraft:ambient.cave"), SelfSoundSource, AdventureSound.Source.MASTER, 1.0f, 1.0f)
+// highlight-green
+val sound = Sound(DefaultSoundId("minecraft:ambient.cave"), SelfSoundSource, AdventureSound.Source.MASTER, ConstVar(1.0f), ConstVar(1.0f))
+```
+
+**To get the volume and pitch:**
+Accessing the `volume` and `pitch` now requires a `player` and an optional `interactionContext`.
+
+```kotlin showLineNumbers
+// highlight-red
+val volume: Float = sound.volume
+val pitch: Float = sound.pitch
+// highlight-green
+val volume: Float = sound.volume.get(player, interactionContext?)
```

Comment:
I don't think this is actual valid kotlin code. The `interactionContext?` is not valid